src/control/run_go_tests.sh

```diff
@@ -2,6 +2,59 @@
 ## Run linters across control plane code and execute Go tests
 set -e
 
+function find_build_source()
+{
+	BASE=$PWD
+	while true
+	do
+		path=$(realpath "${BASE}")
+		if [ "${path}" == "/" ]; then
+			break
+		fi
+		test -e "${path}/.build_vars.sh" && echo "${path}/.build_vars.sh" && return
```

Comment:
(style)  line over 80 characters

Reply:
splitting before 80 characters makes it needlessly complicated.